cristin-person/src/main/java/no/unit/nva/cristin/person/update/PersonPatchValidator.java

```diff
@@ -171,6 +172,7 @@ private static void validateNviIfPresent(ObjectNode input) throws BadRequestExce
                 Optional.of(personNvi)
                     .map(PersonNvi::verifiedBy)
                     .map(PersonSummary::id)
+                    .filter(uri -> StringUtils.isNotBlank(uri.toString()))
```

Comment:
Ser ut som den nye jackson versjonen parser id: "" til URI("") istedet for null som knekte test.

Reply:
Det finnes antageligvis mer elegante måter å fikse sjekken på om URI is empty.

Reply:
Det var derfor jeg valgte å kjøre på versjon 2.15 . Hvis nva-commons krever 2.16 måtte vi nok endre her også ja